frontend/src/static/js/v_summary.js

```diff
@@ -681,26 +681,27 @@ window.vSummary = {
       });
     },
 
-    openTabZoomSubrange(userOrig) {
+    openTabZoomSubrange(user, repo, index) {
       // skip if accidentally clicked on ramp chart
       if (drags.length === 2 && drags[1] - drags[0]) {
         const tdiff = new Date(this.filterUntilDate) - new Date(this.filterSinceDate);
         const idxs = drags.map((x) => x * tdiff / 100);
         const tsince = getDateStr(new Date(this.filterSinceDate).getTime() + idxs[0]);
         const tuntil = getDateStr(new Date(this.filterSinceDate).getTime() + idxs[1]);
 
-        this.openTabZoom(userOrig, tsince, tuntil);
+        this.openTabZoom(user, tsince, tuntil, repo, index);
       }
     },
 
-    openTabZoom(userOrig, since, until) {
+    openTabZoom(user, since, until, repo, index) {
       const { avgCommitSize } = this;
-      const user = Object.assign({}, userOrig);
+      const clonedUser = Object.assign({}, user);
```

Comment:
On a second thought, I think the implementation can be further improved.

The idea of this cloned user is to modify the commits within the user without affecting the original users. However, as pointed out by #971, the implementation is not ideal as it is still directly modifying the `property` field of the child component.  To fix it, we can make the `clonedUser` as a computed field inside `v_zoom` as how it is done in #971.

https://github.com/reposense/RepoSense/pull/971/files#diff-64d5324a6164c3a10fbca7a7508041d7L18-L26

So, there are two choices now. fix this issue inside this PR as we are already touching it or fix it inside #971 first as it is more related.

Reply:
@eugenepeh Your thought?

Reply:
I think its better if we let #971 handle it, as it actually works on this aspect of the issue. We can merge this one, after it.

Reply:
Fixing in #971 would be better in that case, thanks @fzdy1914